requirements.txt

```diff
@@ -35,7 +35,7 @@
 #   deactivate
 #
 
-Django>=1.9,<1.9.99
+Django>=1.9,<1.10
```

Comment:
I think the best way would be using the [compatible release operator](https://www.python.org/dev/peps/pep-0440/#compatible-release) here actually. Strangely no one on that mailing list thread realized you can use it:

    Django~=1.9.0

Is equivalent to asking for Django 1.9.*